internal/provider/resource_port_test.go

```diff
@@ -2,65 +2,57 @@ package provider
 
 import (
 	"fmt"
+	"os"
 	"testing"
 
 	"github.com/PacketFabric/terraform-provider-packetfabric/internal/testutil"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
-func hclPort(description, accountUUID, speed, media, subscriptionTerm, pop, zone string, autoneg, nni bool) (hcl string, resourceName string) {
+func hclPort(description, media, speed, pop, subscriptionTerm string) (hcl string, resourceName string) {
 	hclName := testutil.GenerateUniqueResourceName()
 	resourceName = "packetfabric_port." + hclName
-	hcl = fmt.Sprintf(`
-	resource "packetfabric_port" "%s" {
-		description       = "%s"
-		account_uuid      = "%s"
-		speed             = "%s"
-		media             = "%s"
-		subscription_term = "%s"
-		pop               = "%s"
-		zone              = "%s"
-		autoneg           = %t
-		nni               = %t
-	}`, hclName, description, accountUUID, speed, media, subscriptionTerm, pop, zone, autoneg, nni)
+	hcl = fmt.Sprintf(testutil.RResourcePort, hclName, description, media, pop, speed, subscriptionTerm)
 	return
 }
 
 func TestAccPort(t *testing.T) {
 	testutil.SkipIfEnvNotSet(t)
 
-	description := testutil.GenerateUniqueName(testPrefix)
-	pop, zone, err := testutil.GetPopAndZoneWithAvailablePort("1Gbps", "LX")
-	if err != nil {
-		t.Fatalf("Unable to find pop and zone with available port: %s", err)
-	}
+	description := os.Getenv(testutil.PF_PORT_DESCR_KEY)
```

Comment:
Could we use unique generated names here? Tests will create multiple ports during one run. It would be cool to be able to tell the difference in the API logs :) Also, running tests in parallel will be even messier.